Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs

```diff
@@ -139,9 +136,35 @@ public void Initialize(Material skyboxMat, Light dirLight, AnimationClip skyboxA
             RenderSettings.fog = true;
         }
 
+        bool useRemoteSkyboxSettings = featureFlagsCache != null && featureFlagsCache.Configuration.IsEnabled("alfa-skybox-settings");
+
+        int defaultTime = SecondsInDay / 2;
+
+        if (!useRemoteSkyboxSettings)
+        {
+            ApplySkyboxState(defaultTime);
+            return;
+        }
+
+        if (!featureFlagsCache.Configuration.TryGetJsonPayload("alfa-skybox-settings", "settings", out SkyboxSettings skyboxSettings))
+        {
+            ApplySkyboxState(defaultTime);
+            return;
+        }
+
+        Speed = skyboxSettings.speed;
+        ApplySkyboxState(skyboxSettings.time);
```

Comment:
I think you can improve this a tiny-little-bit, removing the 2 exit points and make it easier to read IMHO:

```c#
int defaultTime = SecondsInDay / 2;

if (useRemoteSkyboxSettings && featureFlagsCache.Configuration.TryGetJsonPayload("alfa-skybox-settings", "settings", out SkyboxSettings skyboxSettings))
{
    Speed = skyboxSettings.speed;
    defaultTime = skyboxSettings.time;
}

ApplySkyboxState(defaultTime);
```

Reply:
applied part of the suggestion! thank you!